OWNERS_ALIASES

```diff
@@ -17,17 +17,11 @@ aliases:
     - vpavlin
     - yanniszark
   wg-manifests-leads:
```

Comment:
Yes, i am missing there, even though  i am in the google acls as wg-manifests leads. Please add me.